public/data/Atharv0003.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Your Name",
+  "location": "Your Location",
+  "bio": "Your Bio should be 20-30 words not more then that",
+  "avatar": "https://github.com/Atharv0003.png",
+  "portfolio": "Your Portfolio URL or Github URL",
+  "skills": ["Your Skill 1", "Your Skill 2", "..."],
```

```suggestion
{
  "name": "Your Name",
  "location": "Your Location",
  "bio": "Your Bio should be 20-30 words not more then that",
  "avatar": "https://github.com/Atharv0003.png",
  "portfolio": "Your Portfolio URL or Github URL",
  "skills": ["Your Skill 1", "Your Skill 2", "..."],
```

You have to fill the details @Atharv0003